.vortex/CLAUDE.md

```
@@ -26,7 +26,7 @@ vortex/
 ## .vortex Directory Structure
 
 ### .vortex/docs/
-- **Docusaurus-based documentation** published to https://vortex.drevops.com
+- **Docusaurus-based documentation** published to https://www.vortextemplate.com
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)